lib/providers/models/explore_card_filters_model.dart

```diff
@@ -51,4 +70,25 @@ class ExploreCardFiltersModel extends ChangeNotifier {
     _selectedKeyword = selectedKeyword;
     notifyListeners();
   }
+
+  Input$UserSearchInput toUserSearchInput(int maxResultsCount) {
+    return Input$UserSearchInput(
+      countryTextIds:
+          _selectedCountries.isEmpty ? null : _selectedCountries.toList(),
+      expertisesTextIds:
+          _selectedExpertises.isEmpty ? null : _selectedExpertises.toList(),
+      maxResultCount: maxResultsCount,
+      languagesTextIds:
+          _selectedLanguages.isEmpty ? null : _selectedLanguages.toList(),
+      offersHelp:
+          (_selectedUserTypes.contains("Mentor") || _selectedUserTypes.isEmpty)
+              ? Enum$UserSearchFieldPreference.isTrue
+              : Enum$UserSearchFieldPreference.isFalse,
+      searchText: selectedKeyword,
+      seeksHelp:
+          (_selectedUserTypes.contains("Mentee") || _selectedUserTypes.isEmpty)
+              ? Enum$UserSearchFieldPreference.isTrue
```

Comment:
When I have nothing selected for the "User types" dropdown, it is setting both mentor and mentee preferences to `isTrue`. This means that it will only return users who are both mentors and mentees, which is a small percentage of the users. For example, only 6 users who speak English fulfill this condition in my local DB, which has a couple thousand users.

The scope here was probably not super specific on the behavior. 

One option for default values here could be:
- set `offersHelp` to `Enum$UserSearchFieldPreference.isTrue` by default if the user is a mentee (i.e. if the logged in user has `seeksHelp: true`). Also set `seeksHelp` to `Enum$UserSearchFieldPreference.any` in this case.
- if the user is not a mentee, set `seeksHelp` to `Enum$UserSearchFieldPreference.isTrue` and set `offersHelp` to `Enum$UserSearchFieldPreference.any` by default.

Let me know if that makes sense, thank you!

Reply:
I've switched this to:
```
          _selectedUserTypes.contains("Mentor")
              ? Enum$UserSearchFieldPreference.isTrue
              : Enum$UserSearchFieldPreference.any,
```
Setting the filter to counterpart of logged in user type still needs to be done.